setup.cfg

```diff
@@ -13,4 +13,32 @@ flake8-ignore = E133 E203 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503
 log_level = DEBUG
 
 [metadata]
+name= ts-cbp
```

Comment:
Consider adding a one line description and also something like

```
long_description = file: README.rst
```
or whatever your README file is.